src/OpenTelemetry.ClrProfiler.Managed/Configuration/Settings.cs

```diff
@@ -182,6 +182,26 @@ internal static Settings FromDefaultSources()
             return null;
         }
 
+        private static TracesExporter ParseTracesExporter(IConfigurationSource source)
```

Comment:
It seems that instead of the `IConfigurationSource` you should pass a string (it is already a Parse method).

Reply:
I think that the suggestion for the parse method is important for `public` and `internal` methods, but since it's private I'm less concerned about it. I think this can be revisited when testing out custom exporters.

Reply:
> Q.: how this works with the scenario of adding a custom exporter on a given distribution?

It would be possible to add a custom exporter b/c we basically have a hook to the "SDK builder". However, some other env var key would have to be used (or one could hardcode to always add some exporter).

I would rather design a "exporter <-> env var registration" API in the OTel SDK than here.

Sorry for my English but I hope you got the idea 😄